content/packages.md

```diff
@@ -72,10 +193,181 @@ Include Cordova plugins in your Meteor package by using [Cordova.depends](http:/
 
 Read more about using Cordova in the [mobile guide](#XXX).
 
-## NPM packages
+## NPM packages in a Meteor package
 
 Meteor packages can include [NPM packages](https://www.npmjs.com/) to use JavaScript code from outside the Meteor package ecosystem, or to include JavaScript code with native dependencies.
 
-Include NPM packages in your Meteor package by using [Npm.depends](http://docs.meteor.com/#/full/Npm-depends).
+Include NPM packages in your Meteor package by using [Npm.depends](http://docs.meteor.com/#/full/Npm-depends). For example, here's how you could include the `github` package from NPM:
+
+```js
+Npm.depends({
+  github: '0.2.4'
+});
+```
+
+You can compile client-side NPM packages into your package by using the [`cosmos:browserify`](https://github.com/elidoran/cosmos-browserify) package.
+
+### Converting between callbacks and Fibers
+
+Many NPM packages rely on an asynchronous, callback or promise-based coding style. For several reasons, Meteor is currently built around a synchronous-looking but still non-blocking style using [Fibers](https://github.com/laverdet/node-fibers).
+
+Many Meteor APIs, for example collections, rely on running inside a fiber context; Meteor also keeps track of certain collection state on the server so that code can be associated with a particular client. This means you need to do a little extra work to use asynchronous Node code inside a Meteor app. Let's look at an example of some code that won't work, using the code example from the [node-github repository](https://github.com/mikedeboer/node-github):
```

Comment:
I think you should say a little more about what "inside a fiber context" means. It's pretty opaque here.